### PR TITLE
Expose cAdvisor WebUI through Caddy

### PR DESCRIPTION
**Pull request by GuyKh**
Solves #272

### CODE REVIEW

**Comment by keriati** [comment]
Thank you, very nice. Do we a have to add this port also in docker-compose.yml for cadvisor in the expose list?

**Comment by GuyKh** [comment]
It looks like the [docker-compose](https://github.com/stefanprodan/dockprom/blob/master/docker-compose.yml#L84) already exposes port 8080 for cadvisor

**Comment by keriati** [comment]
I mean for caddy -> [ports](https://github.com/stefanprodan/dockprom/blob/23a3e152e4d446bedc959ab4e943597f6d969764/docker-compose.yml#L123)

**Comment by GuyKh** [comment]
> I mean for caddy -> [ports](https://github.com/stefanprodan/dockprom/blob/23a3e152e4d446bedc959ab4e943597f6d969764/docker-compose.yml#L123)

Actually - you're right.
Fixed it

File: caddy/Caddyfile [hunk 1]
```diff
@@ -20,6 +20,14 @@
     reverse_proxy pushgateway:9091
 }
 
+:8080 {
+    basicauth /* {
+        {$ADMIN_USER} {$ADMIN_PASSWORD_HASH}
+    }
+    
+    reverse_proxy cadvisor:9091
```
Comment (keriati) [inline caddy/Caddyfile:28]:
```suggestion
    reverse_proxy cadvisor:8080
```

**Review by nightah — APPROVED** [review]
LGTM.

Thanks for this @GuyKh!

**Comment by nightah** [comment]
I imagine we're going to see a lot of noise due to the 8080 port mapping because that's a very commonly used port but let's see how this goes.

**Pull request merged**